### ISSUE TITLE
[3.00] Disappeared download

### ISSUE THREAD

**Issue by alek47TU** [transmission/transmission#4925]
Hi All

Waited months for this file to complete, overnight notifications show it reached 100% and seeded yet no sign of the file anywhere! 

Not in the original folder, the completed folder or anywhere in the system after a search.

Makes no sense, any clues welcome. Thanks!

![Screen Shot 2023-02-17 at 6 16 51 AM](https://user-images.githubusercontent.com/60360137/219666522-2526ce7d-0165-4c80-b070-7c1999eb00e6.png)

**Comment by ckerr** [comment]
@alek47TU I really need more to go on than this.

- what version of Transmission
- what UI are you using (e.g. transmission-mac, transmission-gtk, transmission-qt, 3rd party UI)
- what OS are you using
- what does Settings say your general download folder is
- what does Settings say your general incomplete folder is
- what does the Inspector / Torrent Properties say the torrent's folder is
- what happens if you 'verify local data', e.g. can Transmission still find the files
- did you have any scripts set to run when seeding was completed

**Comment by ckerr** [comment]
Closed due to lack of response. If this issue persists in 4.0.1 please open a new issue.

**Comment by alek47TU** [comment]
Closed? I didn’t receive an initial reply!!

Problem still open please, waiting on a solution.

Thanks.


Best


Shane

Shane Daly
=========
síocháin  grá  solas

**Comment by ckerr** [comment]
If the problerm still persists in 4.0.1 please file a new ticket **with the information requested above**

**Comment by alek47TU** [comment]
What? There’s no info above your text and the problem STILL PERSISTS.What’s. The. Answer.BestShaneShane Daly=========síocháin  grá  solasOn Feb 22, 2023, at 19:12, Charles Kerr ***@***.***> wrote:﻿
If the problerm still persists in 4.0.1 please file a new ticket with the information requested above

**Comment by GaryElshaw** [comment]
You might want to read what's been posted @alek47TU https://github.com/transmission/transmission/issues/4925

**Comment by alek47TU** [comment]
what version of Transmission 3.00, update downloaded and ready to install after data check completes.

what UI are you using (e.g. transmission-mac, transmission-gtk, transmission-qt, 3rd party UI) Tranmission-mac

what OS are you using OSX 10.14.6

what does Settings say your general download folder is I have set folders where I want them, usually perfect, the missing file were never in either folder.
what does Settings say your general incomplete folder is

what does the Inspector / Torrent Properties say the torrent's folder is The torrent disappeared overnight beforeI opened the ticket, all I have is the completed and seeded notifications, sent in original screen grab.

what happens if you 'verify local data', e.g. can Transmission still find the files About to complete any second, it's found all the current files properly.

did you have any scripts set to run when seeding was completed No, I never run scripts.

It's just weird, after the notifications the file was completely missing, makes no logical sense.


Thanks

Shane

**Comment by alek47TU** [comment]
> You might want to read what's been posted @alek47TU #4925

Thanks, I missed the hyperlink!

**Comment by alek47TU** [comment]
![Screen Shot 2023-02-23 at 1 04 31 PM](https://user-images.githubusercontent.com/60360137/221017858-70f1400e-d7f4-4237-abc8-4c0c03f2f8e1.png)

It's happened again! This time The Batman, same as before; completed and seeded notifications then the file is completely missing.

Download and completed folders appear normal and correct.

Also, please see new grab, several of the files previously waiting to complete are suddenly finished? But when I try to play them they're corrupt and need indexes fixing. Something's gone wrong somewhere!

**Comment by alek47TU** [comment]
Some of the files in the above grab are missing, some are still in the download folder of which some play and some are corrupt fyi. Thanks.

**Comment by alek47TU** [comment]
Last update if it helps, 18 files were complete but v3 showed they still had plenty of work to finish, they were in the download folder but did not present as finished and did not auto move to the completed folder. All the current downloads have .part names. Thanks.

**Comment by alek47TU** [comment]
None of the 18 files play, they're all partial downloads but v4 labelled them as complete.

**Comment by ckerr** [comment]
Okay, I just can't. There are like a dozen pirated movies you've got in your screenshot there; I can't give technical support to help you download these.

User banned + issue closed. If anyone else is having this problem on public domain torrents please do open a new issue.

**Comment by ckerr** [comment]
Okay, I just can't. There are like a dozen pirated movies you've got in your screenshot there; I can't give technical support to help you download these. Looking back at the first screenshot now I see it was there too but I didn't notice this.

The project can't give technical support for this ticket.

User banned + issue closed.

If anyone else is having this problem on public domain torrents please do open a new issue.
